frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -137,4 +137,25 @@ export class CreateOfferComponent implements OnInit, OnDestroy {
     const input = this.form.get(inputStringId);
     return !input.valid && input.touched || (this.error && !input.valid);
   }
+
+  areDatesValid(form: FormGroup):  {[key: string]: boolean}  {
+    const startedAt = form.get('startedAt').value;
+    const actionOngoing = form.get('actionOngoing').value;
+    const finishedAt = form.get('finishedAt').value;
+    const constantCoop = form.get('constantCoop').value;
```

Comment:
Not sure if this will work here, but you can try a bit more conside object destructuring:
`const { startedAt, actionOngoing, finishedAt, constantCoop } = form.value;`.
Not particularly important, though.